preview/FS-1068-open-type-declaration.md

```diff
@@ -186,84 +175,46 @@ N(12.0) // Works!
 
 We have resolved to use `open type C`.
 
-See original discussion here: https://github.com/fsharp/fslang-design/issues/352#issuecomment-499146012.  
-
 Issues to do with potential breaking changes do to changes in shadowing led to a change of heart, see https://github.com/fsharp/fslang-design/issues/352#issuecomment-522533863
 
-### Open non-static-classes
-
-In C#, any class or struct can be opened using `using static System.String`, and its static content made available. This can be possibly done in the future, as an extension of this feature.
-
-This would enable some things like this:
-
-```fsharp
-open type System.Numerics.Vector2
-
-let v1 = Vector2(1.0f, 2.0f)
-let v2 = Vector2(1.0f, 2.0f)
-
-Dot(v1, v2) // No need to fully qualify 'Dot'
-```
+See original discussion here: https://github.com/fsharp/fslang-design/issues/352#issuecomment-499146012.  
 
-In the preview only static classes could be opened. Post preview have resolved to allow non-static types to be opened.
+### Open non-static-classes
 
-See original discussion here: https://github.com/fsharp/fslang-design/issues/352#issuecomment-499146012.  
+In 'preview', we could only open static classes and not any class or type. 
 
 Issues to do with potential breaking changes coming from changes in shadowing led to a change of heart, see https://github.com/fsharp/fslang-design/issues/352#issuecomment-522533863
 
-Also, consider C# examples of this kind:
-```csharp
-using System.Numerics;
-using static System.Numerics.Vector3;
-using static System.Numerics.Vector2;
-
-namespace ConsoleApp378
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            var v1 = new Vector3();
-            var result1 = Lerp(v1, v1, 1);
-            var v2 = new Vector2();
-            var result2 = Lerp(v2, v2, 1);
-        }
-    }
-}
-```
+See original discussion here: https://github.com/fsharp/fslang-design/issues/352#issuecomment-499146012.  
```

Comment:
This can just be deleted now, since the updated design will ship as preview, thus obviating this whole section